### ISSUE TITLE
Update month and year for 'Featured Diseases' banner in home page

### ISSUE THREAD

**Issue by monicacecilia** [monarch-initiative/monarch-ui#399]
Update month and year for 'Featured Diseases' banner in home page. It still reads 'June 2020'

<img width="1640" alt="Screen Shot 2021-03-11 at 10 38 35 PM" src="https://user-images.githubusercontent.com/1714621/110902438-c1177500-82ba-11eb-8b7a-d5e31d24a275.png">

**Comment by monicacecilia** [comment]
@nicolevasilevsky, could we kindly ask for a status update? Wondering whether you can help us with either updating or curating a list of diseases to display.

**Comment by nicolevasilevsky** [comment]
@monicacecilia of course, happy to help! What exactly is needed?

**Comment by nicolevasilevsky** [comment]
pinging @monicacecilia, see above- thanks!

**Comment by monicacecilia** [comment]
We are thinking of a two fork approach: 
- every quarter or so, we could get 5 new disease added to this carousel (and retire the previous ones)
- every time that is appropriate, we pick diseases 'of the month'

How would this work with your curation schedule, @nicolevasilevsky ?

THANK YOU!

**Comment by monicacecilia** [comment]
Self reminder (for team): https://www.healthline.com/health/directory-awareness-months

(h/t @kevinschaper)

**Comment by nicolevasilevsky** [comment]
Hi @monicacecilia -
how is this: see [spreadsheet](https://docs.google.com/spreadsheets/d/1zuCNTgpn2zqxnN27hBiM6f8XNRriQLgJ9qIPwgGNwWs/edit#gid=1553753231)

I created a list of the disease awareness days based on the list you shared above, and pruned it down, and then started annotating each day with the corresponding Mondo disease and ID. I double checked the disease pages in Monarch to make sure these diseases would be good examples to highlight.

If this on the track, I can finish curating this list and then you'll have a list of over 100 diseases to chose from to highlight each quarter.

Note, if you scroll down, I annotated the diseases in May and June, so you have suggested diseases to highlight this quarter.

cc @jmcmurry

**Comment by monicacecilia** [comment]
@nicolevasilevsky - I forgot to answer: YES! This is perfect, and we are so grateful for your curation magic! Let's finish the list, please! 🌸

**Comment by monicacecilia** [comment]
@jen-martin I added a 'Previously featured' tab to that spreadsheet so we can document the ones that have already rotated. Thank you! 🌷

**Comment by nicolevasilevsky** [comment]
sounds good. Looks like I curated some diseases for May and June. I'll add this to my to do list to finish it before July.

**Comment by nicolevasilevsky** [comment]
> If we want to slightly automate the process, we can use the mondo_ids.txt file to put together a bigger list of featured diseases and repurpose the date field to be the future date (e.g., May 2021 or May-July 2021) in which to display the featured disease.

this sounds like a good approach. I will aim to finish curating this list before the end of June so we'll have new diseases ready for July and the rest of the year.

**Comment by monicacecilia** [comment]
> If we want to slightly automate the process, we can use the mondo_ids.txt file to put together a bigger list of featured diseases and repurpose the date field to be the future date (e.g., May 2021 or May-July 2021) in which to display the featured disease.

Thanks @jen-martin, this is a very good idea. Is there a 'place' of documentation where we can keep this for posterity? Perhaps at https://github.com/monarch-initiative/monarch-ui/blob/master/README.md?  I would like to close this ticket now that #410 is merged.

**Comment by jen-martin** [comment]
I've created ticket #417 to work on semi-automating the diseases-of-the-month diseases from Nicole's spreadsheet.

@monicacecilia : I've added a note the the readme (https://github.com/monarch-initiative/monarch-ui/blob/master/README.md#monarch-uipublic) referencing the "mondo_ids.txt" file which drives the Featured Diseases viewer. Let me know if you'd like more details added to the readme. I also wrote a note in #417 to be sure to document any changes to this file in the readme when that ticket is complete.
